src/common/cvt.cpp

```diff
@@ -158,6 +177,12 @@ class DummyException {};
 static const double eps_double = 1e-14;
 static const double eps_float  = 1e-5;
 
+template <typename T>
+constexpr int sign(T value)
```

Comment:
Please use `static`, enclose it in unnamed namespace or move it to a public generic header.